CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# 0.35.1 (2020-06-??)
```

Comment:
Should have a full date of release :)

Reply:
🤦

Reply:
We'll just  write it as "the mystery release" into Crystal's history books :D

Reply:
Should we add a CI job for validating changelog dates... 🤔 😆